src/commands/unlink.ts

```diff
@@ -0,0 +1,31 @@
+import { flags as oclifFlags } from '@oclif/command'
+
+import { CustomCommand } from '../oclif/CustomCommand'
+import appsUnlink from '../modules/apps/unlink'
+
+export default class Unlink extends CustomCommand {
+  static description = 'Unlink an app on the current directory or a specified one'
+
+  static examples = ['vtex unlink', 'vtex unlink vtex.service-example@0.x']
+
+  static flags = {
+    help: oclifFlags.help({ char: 'h' }),
+    all: oclifFlags.boolean({ char: 'a', description: 'Unlink all apps', default: false }),
+  }
+
+  static args = [
+    { name: 'appId', required: false },
+    { name: 'ithAppId', required: false, multiple: true },
```

Comment:
I think in order for this to work we'll have to use `static strict = false`
```
For variable length arguments, disable argument validation with static strict = false on the command.
https://oclif.io/docs/args
```

Reply:
Nice catch!